integrations/Klaviyo/browser.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+import { extractCustomFields } from "../../utils/utils";
+
+class Klaviyo {
+  constructor(config) {
+    this.publicApiKey = config.publicApiKey;
+    this.apiKey = config.privateApiKey;
+    this.listId = config.listId;
+    this.consent = config.consent;
+    this.smsConsent = config.smsConsent;
+    this.name = "KLAVIYO";
+    this.CONFIG_MEMBERSHIP = "Membership";
+    this.CONFIG_SUBSCRIBE = "Subscribe";
+  }
+
+  init() {
+    logger.debug("===in init Klaviyo===");
+    ScriptLoader(
+      "klaviyo-integration",
+      `https://static.klaviyo.com/onsite/js/klaviyo.js?company_id=${this.publicApiKey}`
+    );
+  }
+
+  isLoaded() {
+    logger.debug("===in isLoaded Klaviyo===");
+
+    return !!window._learnq;
```

Comment:
return !!(window._learnq&& window._learnq.push !== Array.prototype.push); 
Can we use like this? 
Same for isReady()

Reply:
Changed